README.md

```diff
@@ -46,9 +48,18 @@ npm run dev
 
 ```
 
+### Env variables:
+
+-   PORT - port that server is listening on.
+-   REDIS_HOST - host for redis instance. Can be with `redis://` or without.
+-   REDIS_PORT - port for redis instance.
+-   REDIS_PASSWORD - password for redis. Running redis by docker-compose, there's no password. Running by https://redislabs.com/try-free/ you have to pass this variable.
+-   REDIS_ENDPOINT_URI - redis password, port and host encoded in redis uri take precedence over other environment variables. Can be with `redis://` or without.
```

Comment:
there is no password inside this endpoint

Reply:
Host and a port encoded in redis uri take precedence over other environment variable. 
Can be with `redis://` or without.